src/main.py

```diff
@@ -189,5 +268,9 @@ def generate_samples(args, cc):
     elif args.task == 'generate':
         cc = initialize_settings(args)
         generate_samples(args, cc)
+
+    elif args.task == 'ensemble-generator':
+        cc = initialize_settings(args)
+        ensemble_optimization(args, cc)
```

Comment:
I think that we should add a new task called 'test-classifiers' to allow us to load the discriminators and evaluate their performance by using most voted.
It should be something similar to 'generate' but loading the classifiers (discriminators+classification layer) and testing in on a given data set and a number of samples.